apps/proposer/src/routes/proposer.mjs

```diff
@@ -6,30 +6,109 @@ Some transactions are so simple that, we don't split out a separate service
 module but handle the entire request here.
 */
 import express from 'express';
-import logger from '../../common-files/utils/logger.mjs';
+import {
+  getCurrentProposer,
+  getProposers,
+  registerProposer,
+  updateProposer,
+  unregisterProposer,
+  changeCurrentProposer,
+  withdrawBond,
+} from '../services/proposer.mjs';
 
 const router = express.Router();
 
-router.post('/offchain-transaction', async (req, res) => {
-  const nf3 = req.app.get('nf3');
-  logger.debug(`Proposer/offchain-transaction endpoint received POST`);
-  logger.debug(`With content ${JSON.stringify(req.body, null, 2)}`);
-  const { transaction } = req.body;
+/**
+ * TODO this currently matches two paths (+ another one with the query) since there is some functionality
+ * lacking on optimist itself, but we should have these three routes planned
+ * */
+router.get(['/', '/current'], async (req, res, next) => {
+  try {
+    const { id } = req.query;
+
+    const proposer = id ? await getCurrentProposer() : await getCurrentProposer();
```

Comment:
Regardless of the result of ternary it calles the same `getCurrentProposer` method.

Reply:
Yes, this is because there's no `service` for `id`... Yet. This route will give you the same answer whether you ask for current proposer, for this service's proposer, or for a specific proposer. I left the ternary so it's easier later to just replace one of the functions with the correct function that accepts the `id` param